pages/_app.tsx

```diff
@@ -61,18 +58,13 @@ function MyApp({ Component, pageProps }: MyAppProps) {
   return getLayout(
     <>
       <Head>
-        <title>{title}</title>
-        <meta name="description" content={description} />
-        <link rel="icon" href="/favicon.ico" />
-
-        <meta property="og:title" content={title} />
-        <meta property="og:description" content={description} />
-        <meta property="og:image" content={previewImage} />
-
-        <meta name="twitter:title" content={title} />
-        <meta name="twitter:description" content={description} />
-        <meta name="twitter:image" content={previewImage} />
-        <meta name="twitter:card" content="summary_large_image" />
+        <link key="favicon" rel="icon" href="/favicon.ico" />
+        <meta key="twitter:card" name="twitter:card" content="summary_large_image" />
+        <meta key="twitter:url" name="twitter:url" content="https://www.mycovidstory.ca" />
```

Comment:
twittler:url and og:url are like canonicals, they should be the link to the actual page, not the main home page!

Reply:
Ah shit. Are they strictly necessary? I looked at a couple of guides and seems borderline. This'll need some router action to do.

Reply:
The more important one would be the actual canonical tag which is currently missing, but it also definitely needs to have the fully qualified URL. 

Maybe leave og:url and twitter:url off for now, and we can create an issue for all 3 that require it?

Reply:
Canonical is only for duplicate content. Having the homepage canonicalized is kind of important (and it is now in this PR), but beyond that I don't think it's worth it. 

I'm going to remove `og:url` and `twitter:url` for now and we can revisit.